site/sfguides/src/getting_started_with_native_apps/getting_started_with_native_apps.md

```diff
@@ -114,19 +116,150 @@ Let's explore the directory structure:
+There `src` directory is used to store all of our various source code including stored procedures, user defined functions (UDFs), our streamlit application, and even our installation script `setup.sql`.
 
-`src`
 
-- the `src` directory is used to store all of our various source code including stored procedures, user defined functions (UDFs), our streamlit application, and even our installation script `setup.sql`.
+<!-- ------------------------ -->
+## Upload Necessary Data
+Duration: 2
+
+### Upload provider shipping data 
+
+Now, let's create a database that we will use to store provider's shipping data. This is the data that we will share with the application so that the consumer can enrich their own supply chain data with it when they install the app in their account.
+
+This commands are run by executing the **prepare_data.sh** file. But first we are going to explained its contents.
+
+First the file creates the database, warehouse, schema, and defines the table that will hold the shipping data.
+
+```
+snow sql -q "
+CREATE OR REPLACE WAREHOUSE NATIVE_APP_QUICKSTART_WH WAREHOUSE_SIZE=SMALL INITIALLY_SUSPENDED=TRUE;
+
+-- this database is used to store our data
+CREATE OR REPLACE DATABASE NATIVE_APP_QUICKSTART_DB;
+USE DATABASE NATIVE_APP_QUICKSTART_DB;
+
+CREATE OR REPLACE SCHEMA NATIVE_APP_QUICKSTART_SCHEMA;
+USE SCHEMA NATIVE_APP_QUICKSTART_SCHEMA;
+
+CREATE OR REPLACE TABLE MFG_SHIPPING (
+  order_id NUMBER(38,0), 
+  ship_order_id NUMBER(38,0),
+  status VARCHAR(60),
+  lat FLOAT,
+  lon FLOAT,
+  duration NUMBER(38,0)
+);"
+```
+
+### Upload Consumer Supply Chain Data
+
+In this scenario, consumers will provider their own supply chain data (orders and site recovery data) from their own Snowflake account. The app will use the consumer's data to render graphs representing different aspects of the supply chain.
+
+We'll use the `NATIVE_APP_QUICKSTART_DB` to store the consumer supply chain data.
+
+```
+snow sql -q "
+USE WAREHOUSE NATIVE_APP_QUICKSTART_WH;
+
+-- this database is used to store our data
+USE DATABASE NATIVE_APP_QUICKSTART_DB;
+
+USE SCHEMA NATIVE_APP_QUICKSTART_SCHEMA;
+
+CREATE OR REPLACE TABLE MFG_ORDERS (
+  order_id NUMBER(38,0), 
+  material_name VARCHAR(60),
+  supplier_name VARCHAR(60),
+  quantity NUMBER(38,0),
+  cost FLOAT,
+  process_supply_day NUMBER(38,0)
+);
+
+
+CREATE OR REPLACE TABLE MFG_SITE_RECOVERY (
+  event_id NUMBER(38,0), 
+  recovery_weeks NUMBER(38,0),
+  lat FLOAT,
+  lon FLOAT
+);"
+```
+
+Once we have created the necessary tables in the provider and consumer side, the next step is to load the csv files to the corresponding tables.
+That is accomplished using the `snow object stage copy` command, this uploads it directly to the table associated stage.
+
+```bash
+# loading shipping data into table stage
+snow object stage copy ./app/data/shipping_data.csv @%MFG_SHIPPING --database NATIVE_APP_QUICKSTART_DB --schema NATIVE_APP_QUICKSTART_SCHEMA
```

Comment:
Hot off the presses, this command is now deprecated. You can replace it with `snow stage copy` (just remove `object` for all stage commands). Make sure you have the latest version of Snowflake CLI installed